src/components/top/myft-disengaged-tooltip/main.js

```diff
@@ -5,7 +5,7 @@ import Cookies from 'js-cookie';
 
 const articleAddToMyftButton = document.querySelector('.topper__primary-theme .n-myft-follow-button');
 const headerMyFTLogo = document.querySelector('[data-trackable="my-ft"]');
-const externalReferer = !document.referrer || !(new URL(document.referrer).hostname.endsWith('ft.com'));
+const externalReferer = document.referrer && !(new URL(document.referrer).hostname.endsWith('ft.com'));
```

Comment:
```suggestion
const externalReferer = document.referrer && !(new URL(document.referrer).hostname.endsWith('.ft.com'));
```

otherwise hovercraft.com gets through